content/security/advisory/2017-04-27.adoc

```diff
@@ -19,7 +19,7 @@ The temporary files typically are only on the file system for the duration of a
 This change sets permissions on the temporary files to be readable only by the file owner.
 If a workspace is available, a temporary directory adjacent to the workspace is used instead of the system temporary directory.
 
-As a workaround, setting the system property `java.io.tmpdir` on Jenkins master and agents to a directory only accessible by the Jenkins process's user would prevent this issue.
+As a workaround, setting the system property `java.io.tmpdir` on Jenkins controller and agents to a directory only accessible by the Jenkins process's user would prevent this issue.
```

Comment:
```suggestion
As a workaround, setting the system property `java.io.tmpdir` on the Jenkins controller and agents to a directory only accessible by the Jenkins process's user would prevent this issue.
```

Reply:
I deliberately skipped this one given that agents would also be used without article, but I guess we can do it either way?